frontend/src/store.js

```diff
@@ -10,22 +10,106 @@ const MUTATIONS = Object.freeze({
 
 export default new Vuex.Store({
   state: {
-    students: []
+    students: [],
   },
   mutations: {
     [MUTATIONS.SET_STUDENTS](state, students) {
       state.students = students
-    }
+    },
+
   },
   actions: {
     async getAllStudents({ commit }) {
       const response = await fetch("/api/students")
       const data = await response.json()
       commit(MUTATIONS.SET_STUDENTS, data)
-    }
+    },
+
+    // Update student status
+    async updateStudentStatus({ commit, dispatch }, { studentID, previousStatusString, nextStatusString, updatedBy }) {
+      var success = true;
+
+      // PATCH student
+      const studentRequestOptions = {
+        method: "PATCH",
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({StudentID: studentID, StatusString: nextStatusString})
+      }
+
+      fetch("/api/students/" + studentID, studentRequestOptions)
+        .then(
+          function(response) {
```

Comment:
this is ok, but maybe can you try to use arrow function?

Reply:
Yes, we have refactored to use arrow function :)